Fabric.Identity.API/Management/ClientController.cs

```diff
@@ -170,6 +170,14 @@ public IActionResult Put(string id, [FromBody] Client client)
         [SwaggerResponse(404, typeof(Error), NotFoundErrorMsg)]
         public IActionResult Delete(string id)
         {
+            var client = _documentDbService.GetDocument<IS4.Client>(id).Result;
+
+            if (string.IsNullOrEmpty(client?.ClientId))
```

Comment:
Seems weird to check for s string to be null or empty when we really want to know if the object is null...I know null propagation will have the result in essentially the same thing because we require ClientId, it just reads strangely to me.